servicepulse/intro-editing-messages.md

```diff
@@ -0,0 +1,48 @@
+---
+title:  Editing failed messages
+summary: Describes the concept of editing and retrying messages in ServicePulse
+component: ServicePulse
+reviewed: 2019-07-04
+---
+## About message edit and retry
+
+If a message cannot be successfully retried it is possible to modify that message prior to retrying it. Both the headers and the body of a message can be edited.
+
+## Enabling the feature
+
+The message edit and retry feature is not enabled by default. To enable it, contact [Particular Support(https://particular.net/support) for assistance.
+
+## Editing headers
+
+Headers on all failed messages can be edited using this feature. There are three categories of headers, each with different editing restrictions.
+
+### Locked headers
```

Comment:
Recommend this should be a bullet list to describe the terms, with terms in bold. Under Custom, the 2 current paragraphs can be merged.

Reply:
Or are you wanting me to add here which headers apply to each category, so that THOSE can be in a bullet list below the text which will serve as introductions to each section?

Reply:
I didn't intend on listing the headers here. I think the UI does a good job of showing what can/can't be edited and that is probably documentation enough.